crates/astria-sequencer/src/app.rs

```diff
@@ -2186,4 +2267,90 @@ mod test {
             },
         )
     }
+
+    #[tokio::test]
+    async fn app_cometbft_block_size_commitmemts_() {
+        // ensure that the returned size of the commimtments is what is intended.
+        // we need the size of whatever is being put into the cometBFT block
+        assert_eq!(
+            std::mem::size_of::<GeneratedCommitments>(),
+            64,
+            "expecting the size to be equal to the length of the commitments being appended to \
+             the cometBFT block"
+        );
+    }
+
+    #[tokio::test]
+    async fn app_prepare_proposal_cometbft_max_bytes_overflow_ok() {
```

Comment:
nice, could you add a test for testing `MAX_SEQUENCE_DATA_BYTES_PER_BLOCK` exceeded as well?